test/test_quantized_conv_mkldnn.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import absolute_import
```

Comment:
I don't think we want to split the tests into separate files

Reply:
I removed separate test files,and add test case to test_quantized.py.
Thanks!